exercises/concept/mixed-juices/.meta/exemplar.js

```diff
@@ -12,18 +12,18 @@
  */
 export function timeToMixJuice(name) {
   switch (name) {
-    case 'Pure Strawberry Joy':
-      return 0.5;
-    case 'Energizer':
-      return 1.5;
-    case 'Green Garden':
-      return 1.5;
-    case 'Tropical Island':
-      return 3;
-    case 'All or Nothing':
-      return 5;
-    default:
-      return 2.5;
+  case 'Pure Strawberry Joy':
```

Comment:
here is an example for the changed indentation

Reply:
Thanks! Perfect. I will make sure this is right after we merge @junedev